### PR TITLE
Add exception slides and links

### PR DESCRIPTION
**Pull request by aheejin**
Question: @eholk said he can only attend morning sessions. Should we move this up?

### CODE REVIEW

**Comment by rossberg** [comment]
+1 for moving up, since I want to present as well.

File: 2017/CG-11.md [hunk 1]
```diff
@@ -141,9 +141,11 @@ A list of near by hotels:
             1. [POLL: i64 for `wake` count](https://github.com/WebAssembly/threads/issues/72)
        1. Exception Handling proposal (Heejin Ahn)
             1. [Proposal Repo](https://github.com/WebAssembly/exception-handling)
+            1. [Exception handling scheme in
+toolchain](https://github.com/WebAssembly/tool-conventions/blob/master/EHScheme.md)
+            1. [Slides](https://docs.google.com/a/chromium.org/presentation/d/1Q3jfBS35LVAUZcDo68q0-YKrB8b_UpsxuNgeZvimF0s/edit?usp=sharing)
```
Comment (rossberg) [inline 2017/CG-11.md]: Can't seem to access the slides.

Comment (aheejin) [reply 2017/CG-11.md]: Oh sorry. Fixed it.

**Comment by aheejin** [comment]
@rossberg When are you leaving? (I'm asking because I'm not sure this has to be the first session, or it's ok as long as this is in the morning.)

**Pull request merged**